consensus/safety-rules/src/error.rs

```diff
@@ -47,6 +47,8 @@ pub enum Error {
     NotSafeToTimeout(u64, u64, u64, u64),
     #[error("Invalid TC: {0}")]
     InvalidTimeoutCertificate(String),
+    #[error("Inconsistent Execution Result: {0}, {1}")]
```

Comment:
need more text to explain what the error is, like ordered block info doesn't match executed block info

Reply:
Resolved